builder/bundles.go

```diff
@@ -32,6 +32,7 @@ type buildBundlesConfig struct {
 	UpdateBundle string
 	ContentURL   string
 	VersionURL   string
+	OSName       string
```

Comment:
It doesn't seems like you ever use this variable in the code.

Reply:
It's used in some new code in readBuildBundlesConfig()...

Reply:
It is read, but never used. I sent a PR that completely removes this part of the code since it is not needed anymore.

Reply:
Right, I misunderstood your comment, sorry about that.